### PR TITLE
Dependency Update

### PR DESCRIPTION
**Pull request by boshek**
https://github.com/conbench/conbenchcoms/pull/21 make some changes to the R conbench client. This PR updates that and some other dependencies. In addition, this update some of the reporting code to be inline with that conbenchcoms code. 

fyi @assignUser and @raulcd

### CODE REVIEW

File: .github/workflows/performance-release-report.yml
Comment (assignUser) [inline .github/workflows/performance-release-report.yml:1]: It looks like you changed linenedings or some other whitespace manipulation on the whole file could you revert that?

File: performance-release-report/renv/activate.R [hunk 1]
```diff
@@ -2,8 +2,8 @@
 local({
 
   # the requested version of renv
-  version <- "1.0.2"
-  attr(version, "sha") <- "1.0.2"
+  version <- "1.0.3"
+  attr(version, "sha") <- NULL
```
Comment (assignUser) [inline performance-release-report/renv/activate.R:6]: Is that a change from renv? Or why the change to `NULL`?

**Review by assignUser — APPROVED** [review]
Some whitespace issues but looks good otherwise will merge once those are fixed. Thanks for the updates!

**Comment by assignUser** [comment]
Actually looking at the rendered report from CI it looks like something went wrong? 
``` 

</header>

<p>Please set the BASELINE_GIT_COMMIT &amp; CONTENDER_GIT_COMMIT environment variables to the commit hash of the baseline and the release.</p>

</main>
```
image

File: .github/workflows/performance-release-report.yml
Comment (boshek) [reply .github/workflows/performance-release-report.yml:1]: I'd actually love the keep those if possible. I don't think I'd previously run styler on this file so that's from that.

File: performance-release-report/renv/activate.R (hunk 1 above)
Comment (boshek) [reply performance-release-report/renv/activate.R:6]: This comes directly from renv. This activate file gets updated each version.

**Comment by boshek** [comment]
> Actually looking at the rendered report from CI it looks like something went wrong?
> 
This is actually as intended right now. I wasn't able to give those two variables defaults so the CI report just ended up being this. It is probably worth figuring out at some point but maybe not in this PR?
